gax-java/gax/src/main/java/com/google/api/gax/core/GaxProperties.java

```diff
@@ -100,6 +100,23 @@ public static String getGaxVersion() {
 
   /** Returns the current runtime version */
   private static String getRuntimeVersion() {
```

Comment:
This is pretty complicated logic, can we add some unit tests for it? Since this is a private method, we can test through `getJavaVersion()`.

Reply:
+1 Can we have some tests? Also, some integration tests would be nice. If you want to get fancy, you can do a shell script that installs a couple of versions of Java and tests against those.